packages/react-form/src/utilities.ts

```diff
@@ -32,6 +31,32 @@ export function mapObject<Output>(
   }, {}) as Output;
 }
 
+// Eg: set({a: 1}, ['b', 'c'], 2) // => {a: 1, b: {c: 2}}
+function setObject<T extends Object>(
```

Comment:
Can we add some test coverage for this as well?

Reply:
It is currently just an internal function (non-exported) for `getValue`, but if there is value later, I was thinking we could expose it.